### ISSUE TITLE
image post/upload UX issue regarding drag-and-drop image selection

### ISSUE THREAD

**Issue by sneak** [pixelfed/pixelfed#2574]
I'm a new user of pixelfed.social, and unfortunately it doesn't easily/clearly surface which pixelfed version it's using, so I can't share that.  (Perhaps it could be added to the layout's footer, so that bug reporters can reference it.)

The little plus button in the top menu ("compose") opens a window with three main buttons: New Post, New Story, and New Collection.  I have a photo I'd like to post, as my first post.

1) Dragging it on to this "compose" modal doesn't do what I'd expect (compose a new post using that image).  The browser simply displays the image and changes the location to the file:/// path to my local image.

2) When I click New Post, an OS file selector window appears.  The image I want to post is buried in my filesystem, navigating to it is a pain.  I've already got it handy, I'd like to be able to drop it onto the UI to upload it, but it doesn't appear that that's possible.  (The standard "Choose file..." button for file uploads generally allows for drag-and-drop, but Pixelfed doesn't display one of those.)

**Comment by niccokunzmann** [comment]
Yes, I agree, I am looking forward to this being implemented. Here is a screen shot of the page:
![image](https://user-images.githubusercontent.com/564768/191249168-91f1a05a-1ed5-42a7-9c23-66ee281c6023.png)
I would really like to drag&drop the image anywhere there.

**Comment by sneak** [comment]
It would appear this has been implemented for the gear on the profile avatar (something people do occasionally/rarely) but not for the main photo upload dialog?   Very strange to me that making it easier to do something you do monthly/yearly would be done before making it easier to do *the very main function of the app performed multiple times daily*.
